src/web/lib/hakuneko/frontend@classic-light/pages.html

```diff
@@ -327,6 +327,7 @@
              */
             hideViewer( event ) {
                 this.set( 'selectedMedia', -1 );
+                window.dispatchEvent( new CustomEvent( 'hideViewer', { detail: this.selectedChapter } ) );
```

Comment:
May just set the chapter marker directly:
```javascript
Engine.ChaptermarkManager.addChaptermark( this.selectedChapter );
```
Then no changes in `chapters.html` are required at all